build/charts/antrea/README.md

```diff
@@ -85,6 +85,7 @@ Kubernetes: `>= 1.16.0-0`
 | multicast.igmpQueryInterval | string | `"125s"` | The interval at which the antrea-agent sends IGMP queries to Pods. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". |
 | multicast.multicastInterfaces | list | `[]` | Names of the interfaces on Nodes that are used to forward multicast traffic. |
 | multicluster.enableGateway | bool | `false` | Enable Antrea Multi-cluster Gateway to support cross-cluster traffic. This feature is supported only with encap mode. |
+| multicluster.enablePodToPodConnectivity | bool | `false` | Enable Multi-cluster Pod to Pod connectivity. Cluster Pod CIDRs must be provided to enable Pod to Pod connectivity. |
```

Comment:
I suggest to remove "Cluster Pod CIDRs must be provided to enable Pod to Pod connectivity.", as it is in MC Controller configuration which is never mentioned in the Antrea YAMLs or antrea-agent.conf.

Reply:
Yes, I also hesitated about whether to add this line or not. Removed now.